Makefile

```diff
@@ -309,13 +318,24 @@ protogen_local: go_protoc-go-inject-tag ## Generate go structures for all of the
 	$(PROTOC_SHARED) -I=./p2p/types/proto --go_out=./p2p/types ./p2p/types/proto/*.proto
 
 	# IBC
+	make copy_ics23_proto
```

Comment:
Can you add a check to see if the file is available locally or not? This turns `protogen_local` into a target that makes a network request EVERY time whereas we should only do it when we need it.